assets/js/modules/analytics/components/setup/SetupFormGA4Transitional.js

```diff
@@ -31,11 +26,7 @@ import { Fragment } from '@wordpress/element';
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-import { HIDE_GA4_PROPERTY_SELECT_TOOLTIP } from '../../constants';
```

Comment:
`HIDE_GA4_PROPERTY_SELECT_TOOLTIP` can also be deleted from `assets/js/modules/analytics/constants.js`

Reply:
I've gone ahead and removed this.